src/layout/index.tsx

```diff
@@ -54,6 +55,7 @@ const Layout = ({ children }: { children: React.ReactNode }) => {
           ></meta>
         </Head>
         {!isHomePage && <Navbar />}
+        {!isHomePage && <BossBanner top={12} />}
```

Comment:
if we need this to show up everywhere, why no just add it only here without the `!isHomePage` check and remove it from the `LandingPage`?